src/include/mpir_comm.h

```diff
@@ -93,6 +93,7 @@ enum MPIR_COMM_HINT_PREDEFINED_t {
     MPIR_COMM_HINT_MULTI_NIC_PREF_NIC,  /* ch4:ofi */
     MPIR_COMM_HINT_VCI_IDX_SENDER,      /* ch4 */
     MPIR_COMM_HINT_VCI_IDX_RECEIVER,    /* ch4 */
+    MPIR_COMM_HINT_VCI, /* ch4. Generic hint, useful to select vci_idx, progress thread etc. */
```

Comment:
Isn't this redundant from the two sender/receiver specific hints?

Reply:
In a way, yes. We needed this higher level hint to support OneCCL.

Reply:
I see. Do we have code to ensure the consistency between `MPIR_COMM_HINT_VCI` and the other two hints?